doc/content/gateways/thethingsindoorgateway/_index.md

```diff
@@ -0,0 +1,112 @@
+
+{{< figure src="TTIG.jpeg" alt="{{% ttig %}}" >}}
+
+## Prerequisites
+
+1. User account on {{% tts %}} with rights to create Gateways and API Keys.
+2. The EUI of the gateway. This EUI is a **16 character** value that can be found at the bottom in the WiFi setup screen.
+{{< figure src="TTIG_EUI.png" alt="{{% ttig %}} EUI" >}}
+
+{{< note >}} The EUI of the gateway is **NOT** the WiFi MAC address printed on the back of the gateway, but is derived from the first number that typically is of the form **58A0CBxxxxxx**. It's located on the top of the sticker below the QR code. For example, if that number is 5BA0CB800BE7, insert **FFFE** after the first 6 characters to make it a 16 character Gateway EUI like 5BA0CBFFFE800BE7{{</ note >}}
+
+3. The WiFi password of the {{% ttig %}}, typically printed on the back of the gateway. 
+
+
+## Claiming {{% ttig %}}
+
+{{% ttig %}} is added to The Things Stack via a process called **Gateway Claiming**.
+
+{{< warning >}} Do not register your TTIGs via the regular option of **Adding Gateways**. {{</ warning >}}
+
+### Procedure
+
+Go to **Gateways** in the top menu, and click **Claim Gateway** to reach the gateway claiming page.
+
+Fill the **Gateway EUI**, the **Claim authentication code** (the WiFi Password), the **Gateway ID** and choose the appropriate **Frequency Plan**.
+
+Click **Claim gateway** to finish.
+
+{{< figure src="TTIG_Claim.png" alt="{{% ttig %}} Claiming" >}}
+
+If your inputs are correct, a new gateway will be created and you will be redirected to the gateway overview page of your newly created gateway.
+
+## Connecting {{% ttig %}}
+
+### New Gateways
+
+Press the reset button (small button at the back of the gateway next to the USB-C port) for 5 seconds until the LED blinks rapidly GREEN<->RED for a couple of times.
+
+Hold the SETUP (button at the top of the gateway, next to the LED) for 10 seconds until the LED blinks RED rapidly.
+
+The gateway now exposes a WiFi AP whose SSID is **MINIHUB-xxxxxx** where `xxxxxx` is the last 6 digits of the gateway EUI. The password for this network is printed on the back panel of the device under **WiFi PW**.
+
+After connecting to this network go to *192.168.4.1* using a web browser to access the WiFi config page.
+{{< figure src="TTIG_WiFi_Setup.png" alt="{{% ttig %}} WiFi Setup" >}}
+
+Select the WiFi network by clicking the **+** button on the desired network (and enter the password if it's a secure network).
+
+{{< note >}} {{% ttig %}} does not support enterprise WiFi networks. {{</ note >}}
+
+Select the **Save and Reboot** option. 
+
+If your configuration is correct, 
+  * The gateway will blink GREEN for a few seconds while it connects to this network.
+  * Then, it will blink GREEN<->RED for a few seconds while it connects to the server and fetches the necessary configuration.
```

Comment:
```suggestion
  * Then, it will blink from GREEN to RED and vice versa for a few seconds while it connects to the server and fetches the necessary configuration.
```